microsoft/testsuites/xfstests/xfstests.py

```diff
@@ -299,49 +411,109 @@ def get_xfstests_path(self) -> PurePath:
 
     def set_local_config(
         self,
+        file_system: str,
         scratch_dev: str,
         scratch_mnt: str,
         test_dev: str,
         test_folder: str,
-        test_type: str,
-        fs_type: str,
+        test_section: str = "",
         mount_opts: str = "",
+        testfs_mount_opts: str = "",
+        additional_parameters: Optional[Dict[str, str]] = None,
+        overwrite_config: bool = False,
     ) -> None:
+        """
+        About: This method will create // append a local.config file in the install dir
+        local.config is used by XFStest to set global as well as testgroup options
+        Note:You can call this method multiple times to create multiple sections.
+        The code does not checks for duplicate section names, so that is the users
+        responsibility.
+        Also take note of how options are carried between sectoins, that include the
+        sections which are not going to be run.
+        Recommend going through link:
+        https://github.com/kdave/xfstests/blob/master/README.config-sections
+        for more details on how to use local.config
+        Parameters:
+            scratch_dev (str)   : (M)The scratch device to be used for testing
+            scratch_mnt (str)   : (M)The scratch mount point to be used for testing
+            test_dev (str)      : (M)The test device to be used for testing
+            test_folder (str)   : (M)The test folder to be used for testing
+            file_system (str)   : (M)The filesystem type to be tested
+            test_section (str)  : (O)The test group name to be used for testing.
+                Defaults to the file_system
+            mount_opts (str)    : (O)The mount options to be used for testing.
+                Empty signifies disk target
+            testfs_mount_opts (str): (O)The test filesystem mount options to be used for
+                testing.Defaults to mount_opts
+            additional_parameters (dict): (O)Additional parameters (dict) to be used for
+                testing
+            overwrite_config (bool): (O)If True, the existing local.config file will be
+                overwritten
+        Example:
+        xfstest.set_local_config(
+            scratch_dev="/dev/sdb",
```

Comment:
we can't hardcode disk names here, since they will change the name after reboot.